packages/node_modules/@webex/internal-plugin-ediscovery/src/ediscovery.js

```diff
@@ -286,17 +285,37 @@ const EDiscovery = WebexPlugin.extend({
     options = {...defaultOptions, ...options};
 
```

Comment:
maybe we should the comment here to use the word space instead of report since the cache is now per space

Reply:
good catch, done